routes/index.js

```diff
@@ -5,4 +5,17 @@ router.get("/", (req, res, next) => {
   res.render("index");
 });
 
+// render main page
+router.get("/main", (req, res, next) => {
+  if (req.session.currentUser) {
```

Comment:
Une amélioration : tester si le user est PAS connecté et stopper tout -> autrement la suite

```js
(req, res, next) => {
  if (!req.session.currentUser) return res.redirect('/auth/signin') // 🛑
  
  // otherwise ...
}
```